vmdb/app/controllers/api_controller/policies.rb

```diff
@@ -0,0 +1,24 @@
+class ApiController
+  module Policies
+    #
+    # Policies and Policy Profiles Subcollection Supporting Methods
+    #
+
+    def policies_query_resource(object)
+      return {} unless object
+      (object.class.name == collection_config[:policy_profiles][:klass]) ? object.members : object_policies(object)
```

Comment:
Avoid comparing by class name string.  This can probably be done with `.kind_of?` or `.instance_of?`